docs/style.css

```
@@ -226,33 +250,49 @@ pre::-webkit-scrollbar-thumb:hover { background: var(--mcj-ink-muted); }
   opacity: 1 !important;
 }
 
-/* ── Callouts (Note / Tip / Warning / Info) ─────────────────────────────── */
+/* ── Callouts — editorial left-rail-only, no soft-fill box ──────────────── */
 
+/* Strip the rounded tinted box that ships by default. Replace with a clean
+   left rail in the tone color and slightly larger body text — reads as a
+   newspaper aside / pull-quote, not a generic alert. Tones derive from the
+   design-system semantic status colors. */
 callout,
```

Stylelint (17.12.0)

[error] Unknown type selector "callout" (selector-type-no-unknown)

(selector-type-no-unknown)